products/fuse/get-started.adoc

```diff
@@ -6,7 +6,6 @@ Prerequisites &#38; Required Dependencies Checklist
 
 == Prerequisites section
 1. Java SE Development Kit (JDK), we recommend OpenJDK or Oracle JDK.
-2. JBoss&#174; Developer Studio 9.0
```

Comment:
You should check the other get-started guides to see if this problem is present there too.